src/utils/quotesService.js

```diff
@@ -22,10 +22,14 @@ const roleQuotes = {
 };
 
 export function getQuote(role, action, quoteID) {
-  if (!role || !quoteID || !action) {
+  if (!role || (!quoteID && !action)) {
     return DEFAULT_QUOTE;
   }
 
+  if (!roleQuotes[role][action]) {
+    return;
```

Comment:
The more I think about it it feels more intuitive to actually return _null_ here.